cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4793,4 +4793,28 @@
       Determines if tethering server shouuld be enabled.
     </description>
   </property>
+
+  <property>
+    <name>provisioner.system.properties.gcp-dataproc.compute.request.connectionTimeout</name>
```

Comment:
Typically extensions configurations are not going into the cdap-default.xml.

Also, there are two GCP dataproc provisioners, `gcp-dataproc` and `gcp-existing-dataproc`

Reply:
Also, the naming convention is to have words separated by `.` instead of using camel case. E.g. `connection.timeout.millis`. Moreover, always carry and document the unit of timeout.

Reply:
Done